Corona-Warn-App/src/main/java/de/rki/coronawarnapp/dccticketing/core/check/DccTicketingServerCertificateCheckException.kt

```diff
@@ -9,6 +9,7 @@ class DccTicketingServerCertificateCheckException(
         val message: String
     ) {
         CERT_PIN_NO_JWK_FOR_KID("No matching jwk for required kid"),
-        CERT_PIN_MISMATCH("The SHA-256 fingerprint of leafCertificate is not included in requiredFingerprints")
+        CERT_PIN_MISMATCH("The SHA-256 fingerprint of leafCertificate is not included in requiredFingerprints"),
+        CERT_PIN_UNSPECIFIED_ERR("Unspecified error occurred. Needs further investigation")
```

Comment:
where did you get this error code from ? [Specs](https://github.com/corona-warn-app/cwa-app-tech-spec/blob/46af2e99868fb6a57d686fff066a5254758b6d13/docs/spec/dcc-validation-service.md#checking-the-server-certificate-against-a-set-of-jwks)

Please note that these specific error codes are mapped later on based on other use cases into different error codes , we can not just add new ones without aligning with @mlenkeit

Reply:
what about returning `CERT_PIN_MISMATCH` ? if operation failed for any other reason

Reply:
spec.
adjusted.